src/VisionModel/googlenet.jl

```diff
@@ -0,0 +1,42 @@
+export googlenet
+
+function conv_block(inplanes, outplanes, stride, pad, kernel_size)
+  conv_layer = Chain(Conv(kernel_size, inplanes => outplanes, stride = stride, pad = pad),
+                     BatchNorm(outplanes, relu))
+  return conv_layer
+end
+
+function inception_block(inplanes, out_1x1, red_3x3, out_3x3, red_5x5, out_5x5 out_1x1pool)
+  inception_layer = Chain(conv_block(inplanes, out_1x1, kernel_size=(1,1)),
+                          conv_block(inplanes, red_3x3, kernel_size=(1,1))
+                          conv_block(red_3x3, out_3x3, pad=1, kernel_size=(3,3)),
+                          conv_block(inplanes, red_5x5, kernel_size=(1,1)),
+                          conv_block(red_5x5, out_5x5, pad=2, kernel_size=(5,5)),
+                          MaxPool((3, 3), stride=1, pad=1),
+                          conv_block(inplanes, out_1x1pool, kernel_size=(1,1)))
```

Comment:
The inception block isn't a simple chain. It is several parallel chains that converge using concatenation. I've marked up the screen shot below with each chain in a separate color.

<img width="374" alt="Screen Shot 2020-07-29 at 9 29 29 AM" src="https://user-images.githubusercontent.com/4811350/88813260-53309a00-d17e-11ea-857f-fdda6e596e55.png">

Reply:
I have concatenated the inception layers. But I am confused about the dimension input. In testing, we have used (224, 224, 3, 50) so I used ``dims=3`` to stack it.

Reply:
I used ``dims=1`` and it works fine.